develop/tutorials/articles/230-web-experience-management/xx-liferay-fragments-generator/02-using-the-fragment-generator.markdown

```diff
@@ -0,0 +1,56 @@
+# Using the Fragment Generator
+
+After you setup the generator and create your first project by running
+`yo liferay-fragment` you will have the basic structure created, but there's
```

Comment:
`yo liferay-fragments`, not `yo liferay-fragment`

Reply:
Corrected command to "yo liferay-fragments"